### PR TITLE
feat(database): update typeorm

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
Update typeorm to version 0.3.16

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: backend/.eslintrc.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ module.exports = {
     ],
     // import
     'import/export': 'error',
-    'import/no-deprecated': 'error',
+    // 'import/no-deprecated': 'error',
```
Comment (Mogge) [inline backend/.eslintrc.js:43]: I get tons of these and do not know why.

File: database/entity/0057-clear_old_password_junk/UserContact.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class UserContact extends BaseEntity {
   email: string
 
   @Column({ name: 'email_verification_code', type: 'bigint', unsigned: true, unique: true })
-  emailVerificationCode: BigInt
+  emailVerificationCode: string
```
Comment (Mogge) [inline database/entity/0057-clear_old_password_junk/UserContact.ts:35]: see: https://typeorm.io/entities#column-types

Note about bigint type: bigint column type, used in SQL databases, doesn't fit into the regular number type and maps property to a string instead.

**Review by Elweyn — APPROVED** [review]
Sieht gut aus, danke!
Habe die folgenden Funktionnen getestet:
- dev_rest 
- dev_down
- dev_up
- seed
- sendTransaction
- createContribution
- adminDeleteContribution
- adminDenyContribtuion
- adminConfirmContribution
- sendTransactionLink
- createContributionLink
- redeemContributionLink

File: backend/src/graphql/resolver/util/findUsers.ts [hunk 3]
```diff
@@ -30,27 +34,24 @@ export class UserRepository extends Repository<DbUser> {
...
       }
 
       if (filters.byDeleted !== null) {
-        // filterCriteria.push({ deletedAt: filters.byDeleted ? Not(IsNull()) : IsNull() })
         query.andWhere({ deletedAt: filters.byDeleted ? Not(IsNull()) : IsNull() })
       }
     }
 
-    return query
+    return await query
```
[the remark this answers is not in the document]
Comment (ulfgebhardt) [reply backend/src/graphql/resolver/util/findUsers.ts:47]: Why is lint not complaining about this?

[the remark this answers is not in the document]
Comment (Mogge) [reply backend/src/graphql/resolver/util/findUsers.ts:47]: It is not working without the await

**Pull request merged**